solver-project-url-job/base/cronjob.yaml

```
@@ -0,0 +1,92 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: solver-project-url-job
+  labels:
+    app: thoth
+    component: solver-project-url-job
+spec:
+  schedule: "@daily"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: null
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: solver-project-url-job
+        spec:
+          restartPolicy: OnFailure
+          containers:
+            - image: "quay.io/thoth-station/solver-project-url-job:v0.1.5"
```

as now we have imagestream, lets use that
```suggestion
            - image: "solver-project-url-job:latest"
```

---

ok sounds good!